pages/guide.py

```diff
@@ -74,9 +74,7 @@ def process_post_data(self, **kwargs):
       }):
       enterprise_notification_milestone = get_default_first_notice_milestone_for_feature()
 
-    webdx_usecounter_enum = WebDXFeatureObserver.get_enum_by_web_feature(
-      self.form.get('web_feature')
-    )
+    web_feature = self.form.get('web_feature', '')
```

Comment:
I had almost forgotten that this file still exists - this process should feasibly be handled by a POST to the features API, but I suppose that cleanup is not a high priority. 🙂